pkg/oci/client/client.go

```diff
@@ -90,6 +91,10 @@ func (p *provisionerClient) TenancyOCID() string {
 	return p.cfg.Auth.TenancyOCID
 }
 
+func (p *provisionerClient) Region() string {
```

Comment:
This doesn't match the CCM. It uses `phx` for region and if i'm not mistaken this uses the api region, which is `us-phoenix-1`

Reply:
Ah yes, we use `instance.Region` in the CCM:

> The region that contains the Availability Domain the instance is running in.
> Example: phx